index.js

```diff
@@ -134,9 +142,13 @@ function addTests(transform, testDirectory, test) {
     }
 
     if (transform.renderAsync) {
-      test(transform.name + '.renderAsync()', function () {
+      test(transform.name + '.renderAsync()', function (done) {
         return transform.renderAsync(input, options, locals).then(function (output) {
           checkOutput(output);
+          done();
+        }, function (err) {
+          throw new Error(err);
+          done(new Error(err));
```

Comment:
@ForbesLindesay `done(err)` was not reporting failiures. Try this in `test/index.js`...

``` js
testJStransformer({
  name: 'renderAsync: Expect Failure',
  outputFormat: 'txt',
  inputFormats: ['txt'],
  renderAsync: function (str, options, locals) {
    return new Promise(function(fulfill, reject) {
      reject('FAILURE');
    });
  }
}, resolve(__dirname + '/../example/simple'));
```